website_portal_comment_attachment/security/ir.model.access.csv

```diff
@@ -0,0 +1 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
No new model added in this module.
Access right to the attachment for portal user is managed in code because it is too much complex to manage in ir.model.access.